README.md

```diff
@@ -38,7 +40,18 @@ To create a hard link:
 To remove the link:
 - `hln -u destination`
 
+<a name="limitations"></a>
 ## Limitations
+
+### File System
+
+WARNING: APFS (Apple File System) does not support hard links. If you plan on installing MacOS High Sierra and want to continue using hardlink-osx, you will need to maintain your HFS+ file system.
```

Comment:
It does support hard links, just not directory hard links.

Reply:
It appears to support hard links, but in reality they are clonefiles. If the original file is modified in any way the link is broken (original file gets a new inode, the link retains the old inode), essentially making the link a standalone copy of the original linked file.